### PR TITLE
Upgrade TLS version to 1.3 in BesuCommand.java

### PR DESCRIPTION
**Pull request by bhaskarvilles**
Upgrade TLS version to 1.3 in BesuCommand.java
Signed-off-by: Bhaskar Ram <bhaskar@parrtosec.in>

### CODE REVIEW

**Review by macfarla — COMMENTED** [review]
looks good to me and DCO is happy
there is a .dccache file that got accidentally added, can you remove it :)

File: besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java [hunk 1]
```diff
@@ -3041,7 +3041,7 @@ protected void enableGoQuorumCompatibilityMode() {
 
   public static List<String> getJDKEnabledProtocols() {
     try {
-      SSLContext context = SSLContext.getInstance("TLS");
+      SSLContext context = SSLContext.getInstance("TLS v1.3");
```
Comment (macfarla) [inline besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java:3044]: is "v1.3" equivalent to "1.3"?

File: besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java [hunk 2]
```diff
@@ -3030,7 +3030,7 @@ protected void enableGoQuorumCompatibilityMode() {
 
   public static List<String> getJDKEnabledCypherSuites() {
     try {
-      SSLContext context = SSLContext.getInstance("TLS");
+      SSLContext context = SSLContext.getInstance("TLS 1.3");
```
Comment (frankisawesome) [inline besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java:3033]: Looks like you'll need "TLSv1.3" here
https://docs.oracle.com/javase/7/docs/technotes/guides/security/StandardNames.html#SSLContext

**Review by frankisawesome — CHANGES_REQUESTED** [review]

File: besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java (hunk 2 above)
Comment (macfarla) [reply besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java:3033]: is there a test that we can add to ensure it's a valid value?

**Comment by bhaskarvilles** [comment]
> looks good to me and DCO is happy there is a .dccache file that got accidentally added, can you remove it :)

Done !

**Pull request closed without merging**